tests/Integrations/Custom/Autoloaded/TraceSearchConfigTest.php

```diff
@@ -37,7 +37,7 @@ public function testScenario()
                     'web.request',
                     'web.request',
                     'web',
-                    'web.request'
+                    'GET /simple'
```

Comment:
This should be `GET /simple`, right?

Reply:
With `'DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED' => 'true'` yes.

Reply:
I'm not sure why I would have had to change this then -- I didn't modify anything related to that.

Reply:
@morrisonlevi Did you discover this locally? This test does not appear to be running on CI which would explain why it hasn't shown up before: https://github.com/DataDog/dd-trace-php/blob/master/phpunit.xml#L47

Reply:
Yes, I ran it locally.